### PR TITLE
Update the documentation with the new config paths.

### PR DESCRIPTION
**Pull request by reixd**
Update the documentation with the new config path /etc/jitsi/jicofo used in the .deb files.

### CODE REVIEW

File: resources/install/debian/postinst [hunk 1]
```diff
@@ -67,7 +67,7 @@ case "$1" in
 
         fi
 
-        OLD_JITSI_CONFIG="/usr/share/jicofo/.sip-communicator/sip-communicator.properties"
+        OLD_JITSI_CONFIG="/etc/jitsi/jicofo/sip-communicator.properties"
```
Comment (damencho) [inline resources/install/debian/postinst]: This is not correct as the old config is /usr/share/jicofo/.sip-communicator/sip-communicator.properties, it is used to migrate old config to new one.

**Comment by damencho** [comment]
Thank you for your contribution! Before we can merge this and any future contributions, however we need you to sign our CLA (as an [individual](https://jitsi.org/icla) or a [corporation](https://jitsi.org/ccla)).

**Comment by reixd** [comment]
@damencho I signed the CLA.

**Comment by damencho** [comment]
Merged, thanks. bc89f0be4dc3e59018d8ca16d3cb2bd374034543, 
92f89105453cd2148d3a737c2c52cc80196ae326

**Pull request closed without merging**